assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -117,57 +117,69 @@ export const plotSeriesSelectTable = function (elem,
-            .selectAll('th')
-            .data(columnHeaders)
-            .enter().append('th')
-                .attr('scope', 'col')
-                .text(d => d);
+        .selectAll('th')
+        .data(columnHeaders)
+        .enter().append('th')
+        .attr('scope', 'col')
+        .text(d => d);
 
     table.append('tbody')
+        .attr('class', 'usa-fieldset')
         .selectAll('tr')
         .data(availableParameterCodes)
         .enter().append('tr')
-            .attr('ga-on', 'click')
-            .attr('ga-event-category', 'selectTimeSeries')
-            .attr('ga-event-action', (parm) => `time-series-parmcd-${parm.parameterCode}`)
-            .attr('role', 'option')
-            .classed('selected', parm => parm.selected)
-            .attr('aria-selected', parm => parm.selected)
-            .on('click', function (parm) {
-                if (!parm.selected) {
-                    store.dispatch(Actions.updateIVCurrentVariableAndRetrieveTimeSeries(siteno, parm.variableID));
-                }
-            })
-            .call(tr => {
-                let parmCdCol = tr.append('th')
-                    .attr('scope', 'row');
-                parmCdCol.append('span')
-                    .text(parm => parm.description)
-                    .call(appendTooltip, parm => `Parameter code: ${parm.parameterCode}`);
-                tr.append('td')
-                    .append('svg')
-                    .attr('width', SPARK_LINE_DIM.width.toString())
-                    .attr('height', SPARK_LINE_DIM.height.toString());
-                tr.append('td')
-                    .text(parm => parm.timeSeriesCount);
-                tr.append('td')
-                    .style('white-space', 'nowrap')
-                    .text(parm =>`${config.uvPeriodOfRecord[parm.parameterCode].begin_date} to ${config.uvPeriodOfRecord[parm.parameterCode].end_date}`);
-            });
+        .attr('ga-on', 'click')
+        .attr('ga-event-category', 'selectTimeSeries')
+        .attr('ga-event-action', (param) => `time-series-parmcd-${param.parameterCode}`)
+        .attr('role', 'option')
+        .classed('selected', param => param.selected)
+        .attr('aria-selected', param => param.selected)
+        .on('click', function (param) {
+            if (!param.selected) {
+                store.dispatch(Actions.updateIVCurrentVariableAndRetrieveTimeSeries(siteno, param.variableID));
+            }
+        })
+        .call(tr => {
+
+            let paramSelectCol = tr.append('td')
+                .attr('scope', 'row');
+            paramSelectCol.append('input')
+                .attr('type', 'radio')
+                .attr('id', param => `input-${param.variableID}`)
+                .attr('class', 'usa-radio__input param-select-input')
+                .attr('value', param => `${param.variableID}`)
+                .property('checked', param => param.selected)
+                .attr('checked', param => param.selected);
```

Comment:
Do you need both the attribute and property? I would think you only need the property.  Can you verify? Also the way to set boolean attributes in D3 is to assign null when you want it to be false. So we typically do this:
.attr('checked', param => param.selected ? true : null)

Reply:
Yes, I believe the redundancy is an artifact of some earlier trial and error cycles.